proposals/apple-mdm/base-setup-clix.md

```diff
@@ -0,0 +1,205 @@
+# MDM Base Setup
+
+Note: these can all be namespaced behind something like `apple-mdm` if needed. But I would like to avoid creating an mdm vertical vs. an osquery vertical to create an integrated product.
+
+## APNS
+### Step 1 - Generate CSR
+``` sh
+$ fleetctl setup apns init
+Generating MDM keys at ./
+  fleet-apns-pki.crt ... done
+  fleet-apns-pki.key ... done
+  fleet-apns-push.key ... done
+  fleet-apns-push.csr ... done
+Send the .csr and .crt files to your friendly Fleet representative
+
+Keep the .key files. You will need them to complete this process. 
+
+Fleet will return a signed certificate to you within 1 business day. Use that file with the .key files in the `fleetctl setup apns gen-req` command.
+```
+
+### Step 2 - Generate .req file from .p7
+``` bash
+$ fleetctl setup apns gen-req \
+--p7-file=~/path/to/*.p7 \
+--pki-key=~/path/to/fleet-apns-pki.key \
+--pki-crt=~/path/to/fleet-apns-pki.crt
+Working...
+Generated fleet-apns-push.req file at ./
+
+Visit identity.apple.com to upload this .req file. For more details, follow these instructions: NEED LINK HERE
+
+Apple will generate a .crt file. Use that file with the .key files in the `fleetctl setup apns finalize` command. 
+```
+
+### Step 3 - Upload push..crt and push.key files to Fleet ENV
```

Comment:
This cannot be done via `fleetctl`, an administrator would just need to make sure to pass the environment to the `fleet serve` command (fleet server runs the Fleet server).

One of the requirements is to not store sensitive key material in MySQL, thus we decided to pass as environment variables instead.

Reply:
DUH I knew that. will change.